Kentico.Kontent.Delivery.Abstractions/IDeliveryClientExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Threading.Tasks;
+using Kentico.Kontent.Delivery.Abstractions.ContentItems;
+using Kentico.Kontent.Delivery.Abstractions.ContentTypes;
+using Kentico.Kontent.Delivery.Abstractions.TaxonomyGroups;
+
+namespace Kentico.Kontent.Delivery.Abstractions
+{
+    /// <summary>
+    /// Extra overloads of the <see cref="IDeliveryClient"/> for the consumer's convenience. 
+    /// </summary>
+    public static class DeliveryClientExtensions
+    {
+        /// <summary>
+        /// Gets a strongly typed content item by its codename. By default, retrieves one level of linked items.
+        /// </summary>
+        /// <typeparam name="T">Type of the model. (Or <see cref="object"/> if the return type is not yet known.)</typeparam>
+        /// <param name="client">An instance of the <see cref="IDeliveryClient"/></param>
+        /// <param name="codename">The codename of a content item.</param>
+        /// <param name="parameters">An array that contains zero or more query parameters, for example, for projection or setting the depth of linked items.</param>
+        /// <returns>The <see cref="IDeliveryItemResponse{T}"/> instance that contains the content item with the specified codename.</returns>
+        public static async Task<IDeliveryItemResponse<T>> GetItemAsync<T>(this IDeliveryClient client, string codename, params IQueryParameter[] parameters)
+        {
+            return await client.GetItemAsync<T>(codename, parameters);
```

Comment:
You can [elide these calls](https://blog.stephencleary.com/2016/12/eliding-async-await.html) since result of the task is not needed after the call no operations are performed before the call.

Reply:
true. great article, I never realized that :)
I'll incorporate the change

Reply:
resolved in c993a9bd22f337b003b5ffcd968e2017a1827e0b